lib/diagnostic.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = true; // replace null with your response
 //
 
 // Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = ['alderaan', 'deathStar', 'fire'];
 //
```

Comment:
We did not explicitly cover this in class, but the answer should actually be just `deathStar` and `fire`. The reason that `alderaan` would not have to `laserQuality` is that `alderaan` is just a variable representing a string that is totally separate from the `deathStar` object completely.